src/components/CouponDetails/constants.js

```diff
@@ -7,50 +7,78 @@ export const COUPON_FILTER_TYPES = {
   redeemed: 'redeemed',
 };
 
-export const COUPON_FILTER_OPTIONS = {
-  unassigned: 'Unassigned',
-  unredeemed: 'Unredeemed',
-  partiallyRedeemed: 'Partially redeemed',
-  redeemed: 'Redeemed',
-};
-
-export const ACTION_LABELS = {
-  remind: 'Remind',
-  assign: 'Assign',
-  revoke: 'Revoke',
+export const COUPON_FILTERS = {
+  unassigned: {
+    label: 'Unassigned',
+    value: COUPON_FILTER_TYPES.unassigned,
```

Comment:
Is there a reason to not just define this directly? We aren't using the Types in any other spot anymore and it's just a single word each.

As far as making it easier for internationalization, neither really makes it any more difficult or easier

Reply:
Yeah, I originally defined them directly, and then when I realized I wasn't going to get rid of `COUPON_FILTER_TYPES` entirely, I changed it to this.